src/main/java/com/spring/studentmanagement/controllers/UserController.java

```diff
@@ -19,13 +18,19 @@
 @RequiredArgsConstructor
 public class UserController {
 
-    private final UserRepository userRepository;
-
+    private final UserService userService;
 
     @GetMapping
     public String getUserView(Model model) {
-        final List<AppUser> users = this.userRepository.findAll();
+        final List<AppUser> users = this.userService.findAllUsers();
         model.addAttribute("userList", users);
         return "users";
     }
+
+    @PostMapping("/delete")
```

Comment:
Get, Post, Put, Delete etc sunt metode/ verbe http. Daca folosim o metoda de stergere, este redundant sa scriem in calea  URL-ului "/delete", "save" etc, deoarece aceasta operatiune este deja specificata de catre metoda http.